pennylane/transforms/mitigate.py

```diff
@@ -318,6 +318,48 @@ def richardson_extrapolate(x, y):
     return poly_extrapolate(x, y, len(x) - 1)
 
 
+def exponential_extrapolate(x, y, asymptote=None, eps=1.0e-6):
+    r"""Extrapolate to the zero-noise limit using an exponential model (:math:`Ae^{Bx} + C`). This
+    is done by linearizing the data using a logarithm, whereupon a linear fit is performed. Once
+    the model parameters are found, they are transformed back to exponential parameters.
+
+    Args:
+        x (Array): Data in x axis.
+        y (Array): Data in y axis such that :math:`y = f(x)`.
+        asymptote (float): Infinite noise limit expected for your circuit of interest (:math:`C`
+        in the equation above). Defaults to 0 in the case an asymptote is not supplied.
```

Comment:
```suggestion
        asymptote (float): Infinite noise limit expected for your circuit of interest 
            (:math:`C` in the equation above). Defaults to 0 in the case an asymptote is not supplied.
```


This was rendering weirdly without the extra indentation

Reply:
good catch! done in https://github.com/PennyLaneAI/pennylane/pull/5972/commits/5e7867388ab0bfad6e07ff1437676f3a21e71462.